src/pages/patterns/lead-space.mdx

```diff
@@ -0,0 +1,240 @@
+
+When designed and implemented effectively, the lead space can be used to engage visitors immediately in one of the most important moments in the user’s journey through a complex web environment. It is the first section below the masthead and can include a stylized background, headline, supporting message and call to action to accommodate user needs.
+
+</PageDescription>
+
+<br/>
+
+<Row>
+<Column colMd={8} colLg={8} >
+
+![Lead space example](./images/lead-space/lead-space-example-1-768-4.jpg)
+
+</Column>
+</Row>
+
+
+<AnchorLinks>
+
+<AnchorLink>Anatomy</AnchorLink>
+<AnchorLink>Layout</AnchorLink>
+<AnchorLink>Height standards</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Imagery</AnchorLink>
+<AnchorLink>Call to action</AnchorLink>
+<AnchorLink>Pattern in use</AnchorLink>
+<AnchorLink>Related components and patterns</AnchorLink>
+
+</AnchorLinks>
+
+## Anatomy 
+
+Here are some of the elements that are commonly used and currently available. As we continue to build and offer additioanl options, we will share new designs on this page.
+
+![Anatomy of the lead space](./images/lead-space/LeadSpace_Left_Heading06_Anatomy.jpg)
+
+## Layout
+
+### Best practices
+
+Here are some best practices when it comes to layout in lead space:
+
+- Headline should orient the user on what the page is about.
+- The copy below the headline should have a clear relationship to the headline.
+- Buttons are positioned after the headline and the copy. 
+- Image should be relevent to the content of the page. For more details, see the [Imagery section](#imagery).
+
+For actual implementations, please see the component pages linked [below](#related-components-and-patterns).
+
+### Content alignment
+
+On IBM.com pages, there are two layout options&mdash;left-aligned and center-aligned. Follow these guidelines to select the right layout for your page.
+
+1. Align to the primary content of the given page.
+
+2. When there is a background image in the lead space, using left-aligned layout is recommended regardless of the primary content alignement.
+
+<Row>
+<Column colMd={4} colLg={4} >
+
+![Lead space content alignment visual](./images/lead-space/lead-space-content-alignment-example-1-1.png)
+
+</Column>
+<Column colMd={4} colLg={4} >
+
+![Lead space content alignment visual](./images/lead-space/lead-space-content-alignment-example-1-2.png)
+
+</Column>
+<Column colMd={4} colLg={4} >
+
+![Lead space content alignment visual](./images/lead-space/lead-space-content-alignment-example-1-3.png)
+
+</Column>
+</Row>
+
+## Height standards
+
+The size of the lead space usuall reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. TO check out the available lead space heights, see the lead space component pages linked [below](#related-components-and-patterns).
```

Comment:
```suggestion
The size of the Lead space usually reflects the intent of the page and sets the right expectation for the user. Choose from these heights for the different purposes and use cases. To check out the available Lead space heights, see the Lead space component pages linked [below](#related-components-and-patterns).
```

Reply:
Addressed separately in this commit: https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/729/commits/65d22c3886ec45d20c984d311f848b002cfc4d4d